sendContactUsEmail.php

```diff
@@ -0,0 +1,216 @@
+                <meta name="viewport" content="width=device-width, initial-scale=1.0">
+                <style>
+                    body {
+                        font-family: Arial, sans-serif;
+                        background-color: #f4f4f4;
+                        margin: 0;
+                        padding: 20px;
+                    }
+                    .container {
+                        background-color: #ffffff;
+                        border-radius: 8px;
+                        box-shadow: 0 0 10px rgba(0,0,0,0.1);
+                        padding: 20px;
+                        max-width: 600px;
+                        margin: 0 auto;
+                    }
+                    .header {
+                        background-color: #007bff;
+                        color: white;
+                        padding: 10px;
+                        text-align: center;
+                        border-radius: 8px 8px 0 0;
+                    }
+                    .content {
+                        padding: 20px;
+                        font-size: 16px;
+                        line-height: 1.5;
+                        color: #333;
+                    }
+                    .footer {
+                        margin-top: 20px;
+                        text-align: center;
+                        color: #999;
+                        font-size: 14px;
+                    }
+                    .footer a {
+                        color: #007bff;
+                        text-decoration: none;
+                    }
+                </style>
+            </head>
+            <body>
+                <div class="container">
+                    <div class="header">
+                        <h2>Thank You for your Contact Us Request</h2>
+                    </div>
+                    <div class="content">
+                        <p>Hello ' . $yourname . ',</p>
+                        <p>Thank you for reaching out to us. We have received your enquiry. One of our team members will get back to you shortly.</p>
+                        <p>Best regards,<br>TechGirlsHub</p>
+                    </div>
+                    <div class="footer">
+                        <p>&copy; 2024 Your Website. All rights reserved.</p>
+                        <p><a href="https://techgirlshub.co.za">Visit our website</a></p>
+                    </div>
+                </div>
+            </body>
+            </html>
+            ';
+
+            // Send the no-reply email to the user
+            $mail->setFrom('no-reply@techgirlshub.co.za', 'TechGirlsHub - No Reply'); // Set no-reply address
+            if ($mail->send()) {
+                $_SESSION['status'] = "Your enquiry has been submitted!";
+                header("Location: {$_SERVER['HTTP_REFERER']}");
+                exit(0);
+            } else {
+                $_SESSION['status'] = "Feedback Response could not be sent to the user. Mailer Error: {$mail->ErrorInfo}";
```

Comment:
yours should not include feedback

Reply:
Reads ---> "Contact Us Enquiry could not be sent."